homeassistant/components/light/vera.py

```diff
@@ -7,9 +7,13 @@
 https://home-assistant.io/components/light.vera.html
 """
 import logging
+import time
+
 from requests.exceptions import RequestException
 from homeassistant.components.switch.vera import VeraSwitch
 
+from homeassistant.components.light import ATTR_BRIGHTNESS
+
 REQUIREMENTS = ['https://github.com/balloob/home-assistant-vera-api/archive/'
```

Comment:
This would have to be upgraded after your stuff gets merged. As you can see it points now at my repo because I couldn't get my PR merged that added setup.py. If you want, you can fork it and we can have it point at your fork.